Assets/Scripts/ManagerScripts/CountdownManager.cs

```diff
@@ -9,38 +9,55 @@ public class CountdownManager: MonoBehaviour
         public TMP_Text countdownText;
         public GameObject countdownUI;
         public AudioSource countdownSound;
-        private bool _shouldCountTime;
+        public bool shouldCountTime;
         private bool _soundPlayed;
+        private bool _gameStart;
+        
 
         private void Start()
         {
-            _shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
+            shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
```

Comment:
I just realized that we don't need this state variable. CountdownManager shouldn't even be in the main menu scene in the first place

Reply:
> I just realized that we don't need this state variable. CountdownManager shouldn't even be in the main menu scene in the first place
Is this because we're taking it out of the Canvas and making it its own manager? will test that it doesn't need this, but this fixed our bug last time where the countdown was starting in the main menu